### PR TITLE
Hide ask and capture

### PR DESCRIPTION
**Pull request by Meilin77**
### What have I done?

 * Hide the buttons for rogueagent from recruiter
 * Based on the PR  #36 , so maybe check that first

### CODE REVIEW

**Review by mrPaintMan — APPROVED** [review]

File: Project/core/src/main/java/View/screen/GameScreenComponents/AskButton.java [hunk 1]
```diff
@@ -47,6 +52,11 @@ public Actor hit(float x, float y, boolean touchable) {
 
     @Override
     public void draw(Batch batch, float parentAlpha) {
+        this.player = gameController.getGame().getCurrentPlayer();
+        if (player.getClass().equals(RougeAgent.class)){
+            super.draw(batch, parentAlpha); // Important
```
Comment (Dustfoxx) [inline Project/core/src/main/java/View/screen/GameScreenComponents/AskButton.java:57]: Same issue as before, buttons invisible but active. need to disable here i think?

File: Project/core/src/main/java/View/screen/GameScreenComponents/CaptureButton.java [hunk 2]
```diff
@@ -43,6 +48,11 @@ public void changed(ChangeEvent event, Actor actor) {
 
     @Override
     public void draw(Batch batch, float parentAlpha) {
-        super.draw(batch, parentAlpha); // Important
+        this.player = gameController.getGame().getCurrentPlayer();
+        // Only draw the button if the current player is a rouge agent
+        if (player.getClass().equals(RougeAgent.class)){
+                super.draw(batch, parentAlpha);
```
Comment (Dustfoxx) [inline Project/core/src/main/java/View/screen/GameScreenComponents/CaptureButton.java:54]: same thing here, needs to disable button if it is not drawn

File: Project/core/src/main/java/View/screen/GameScreenComponents/RevealButton.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            }
+        });
+    }
+
+    @Override
+    public void draw(Batch batch, float parentAlpha) {
+            this.player = gameController.getGame().getCurrentPlayer();
+            if (player.getClass().equals(RougeAgent.class)){
+                super.draw(batch, parentAlpha);
+            }
```
Comment (Dustfoxx) [inline Project/core/src/main/java/View/screen/GameScreenComponents/RevealButton.java:53]: setdisable. i think this exists in a previous verions though? will probably cause issues on main

**Review by Dustfoxx — CHANGES_REQUESTED** [review]

File: Project/core/src/main/java/View/screen/GameScreenComponents/AskButton.java (hunk 1 above)
Comment (Vilmavds) [reply Project/core/src/main/java/View/screen/GameScreenComponents/AskButton.java:57]: Yes, we can add it to issue #40 I suggest.

**Review by Vilmavds — APPROVED** [review]

**Review by Vilmavds — CHANGES_REQUESTED** [review]
I agree with Fox's comments. The buttons should not appear at all when it's the recruiters turn, and if it's the agents turn but the actions is unavailable, it should be disabled.

**Review by Vilmavds — APPROVED** [review]

**Review by Dustfoxx — APPROVED** [review]

**Pull request merged**